0028-web-monetization/0028-web-monetization.md

```diff
@@ -154,13 +156,18 @@ Dispatched every time an ILP packet with a non-zero amount has been fulfilled by
 ```ts
 {
   detail: {
+    // Received amount
     amount: String,
     assetCode: String,
     assetScale: Number
   }
 }
 ```
 
+(TODO: 
```

Comment:
Let's make it clear that these are the receiving amounts then. We could introduce sent amounts addititively in a later update but for now I think we just need to get the spec up to parity with the implementation